setup.py

```diff
@@ -112,7 +112,6 @@ def run(self):
             "stubtest.py",
             "stubgenc.py",
             "stubdoc.py",
-            "stubutil.py",
```

Comment:
One remaining question is "which stubgen modules to compile?"  

Currently, stubutil contains the base classes and utilities for stubgen and stubgenc, however, it's a bit asymmetrical, because only one of the two is compiled with mypyc.

```
            stubutil.so*
            /        \
     stubgenc.py    stubgen.so*
```

The reason for skipping compilation of these is provided in the comment above:  "Skip these to reduce the size of the build".

I'm tempted to skip compilation of stubgen as well, and just compile stubutil:

```
            stubutil.so*
            /        \
     stubgenc.py    stubgen.py
```

Is there any concern about hurting performance of `stubgen` if I do this?

Reply:
I think it's probably fine to skip compiling stubgen. stubtest is already non-compiled, and in my experience it's plenty fast enough. Moreover, for stubgen, performance isn't nearly as much as an issue as with mypy proper or stubtest. Unlike with mypy proper or stubtest, you're unlikely to be running stubgen repeatedly in CI -- you generally generate stubs once, and then you're done.

Reply:
Those were my thoughts as well.  stubgen does perform an analysis of pure python source files (if not disabled), but that will use the compiled mypy modules.